toolkits/nature/packages/nature-header/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@springernature/nature-header",
-  "version": "9.0.0",
+  "version": "9.0.1",
```

Comment:
This is a breaking change to the `brandContext` which means we need a breaking change here